src/guides/wallet.md

```diff
@@ -0,0 +1,164 @@
+
+Accounts always inherit the watch only behavior of their parent wallet. In other words, a watch only wallet will have exclusively watch only accounts while a regular wallet will have only regular accounts. Accordingly, you can't import private keys into a watch only wallet or public keys into regular wallets. If you try to mix and match watch only wallets and keys with hsd, you're gonna have a bad time.
+
+
+### API Authentication
+`hsd` runs as a server and allow you to interact with your wallets via a REST API. It also allows you protect wallets from unauthenticated requests by running the server with the wallet-auth option. Each wallet you create has a token value that must be passed in with each request. Tokens, like accounts and keys, can also be deterministically generated using your HD seed. This means you can change the token on a wallet as often as you'd like.
+
+### Recovery
+By using the HD standards mentioned above, hsd allows one to easily restore or transfer their entire wallet to different wallet implementations. By providing just the mnemonic, one can fully recover their wallet to a fresh instance of `hsd` or any other software that properly implements BIP33, BIP39, and BIP44, like the Trezor hardware wallet.
+
+*Above borrowed from [Wallets and Accounts and Keys, Oh My!](https://bcoin.io/guides/wallets.html) by Daniel McNally*
+&nbsp;\
+&nbsp;\
+
+<h2 id='examples'>Examples</h2>
+
+Command line examples against a local `hsd` server.
+
+To begin, install [hsd]([https://handshake-org.github.io/) and [hs-client](https://github.com/handshake-org/hs-client)
+&nbsp;\
+&nbsp;\
+
+### hs-client: Interact with your wallet
+
+`hs-client` comes with four executables. For the Wallet you will want to use `hsw-cli` and `hsw-rpc`.
+&nbsp;\
+
+* `hsw-cli` - Wallet REST API
+&nbsp;\
+
+* `hsw-rpc` - Wallet RPC interface 
+&nbsp;\
+
+* `hsd-rpc`  - hsd node RPC interface
+&nbsp;\
+
+* `hsd-cli` - hsd node REST API
+&nbsp;\
+
+
+For more info on node commands unrelated to the wallet. [See here](https://handshake-org.github.io/api-docs))
+
+Below you will find short intros to both wallet tools `hsw-cli` and `hsw-rpc` Use them for wallet creation, sending coins, generating addresses, bidding on names, updating resource records for a name, transfering names, etc.. 
+&nbsp;\
+
+
+### Set the network option
+
+You have four choices to chose when setting your `hsd` node's network. The `hs-client` CLI tools also require this option be set for them to know the port with which to connect to your `hsd` node.
+
+* `regtest` Best for local development work.
+* `testnet` Closest simulation to mainnet. Supported by a network of nodes run by other devs.
+* `mainnet` Live production.
+
+You can set the network for your hsd node with the `--network=` option. Or with `$ export HSD_NETWORK=` 
```

Comment:
I would mention setting an environment variable as opposed to just saying to use `export`.